examples/standard.html

```diff
@@ -35,6 +35,7 @@ <h1>Media Chrome Standard Video Usage Example</h1>
         <media-pip-button></media-pip-button>
         <media-fullscreen-button></media-fullscreen-button>
       </media-control-bar>
+      <media-interaction-feedback></media-interaction-feedback>
```

Comment:
I think we've changed up our examples since this. We should put this in the advanced example.